.kokoro/appengine-datastore-flex.cfg

```diff
@@ -0,0 +1,16 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Idea (feel free to ignore): maybe create an `appengine` directory to make things a little easier to find?

Reply:
I was going to add a few tests first to make sure they run as expected. And then worry about duplication and folders.